docs/developer-guide/integrations/users/ldap.md

```diff
@@ -6,86 +6,155 @@ The purpose of this guide is to explain how to configure MapStore to use an LDAP
+If you want users to log in using their LDAP/AD credentials, you can configure MapStore to connect to your LDAP/AD service. In this setup, the relational database will only store resources and accessory data (permissions, attributes, etc.) related to the users from your LDAP/AD service.
 
-Notice that the LDAP storage is read-only. This means that the MapStore User/Groups management UI cannot be used to manage users and groups.
-This makes sense because an LDAP repository is considered an external source and should be managed externally.
+Note that the LDAP storage is read-only. This means the MapStore User/Groups management UI cannot be used to manage users and groups. This is because an LDAP repository is considered an external source and should be managed externally.
 
-If this can create confusion, you can eventually fully disable the UI when using LDAP, by removing the corresponding plugin from the MapStore configuration.
+If this causes confusion, you can disable the UI when using LDAP by removing the corresponding plugin from the MapStore configuration in `localConfig.json`.
 
-The LDAP storage can be configured in two different ways:
+The LDAP storage can be configured in two ways:
 
-* *synchronized* mode
-* *direct connection* mode (experimental)
+* *Synchronized* mode
+* *Direct connection* mode (experimental)
 
 ### Synchronized mode
 
 In *synchronized mode*, user data (users, groups, roles) is read from LDAP on every login and copied on the internal database.
 
 Any other operation, for example getting user permissions on maps, always uses the internal database.
 
-Synchronized mode is faster for normal use, but data may disalign when users are removed from the LDAP repository.
+Synchronized mode is faster for normal use, but data may dis-align when users are removed from the LDAP repository.
 
 In general we suggest to use synchronized mode, since it is the most stable and tested one.
 
+```mermaid
+sequenceDiagram
+    actor User
+    participant MapStore
+    participant LDAP
+    participant InternalDB
+
+    User->>MapStore: Login with credentials
+    MapStore->>LDAP: Check credentials
+    LDAP-->>MapStore: Credentials valid
+    MapStore->>MapStoreDB: Store user and groups
+    MapStore-->>User: Login successful
+```
+
 ### Direct connection mode (experimental)
 
 In direct connection mode, user data is always read from LDAP, for any operation, so there is no risk of misaligned data.
 
-Direct connection is still experimental and not tested in all the possible scenarios, but will hopefully become the standard mode in an early future, because the approach is simpler and avoids most the synchronized mode defects (e.g. misalignments).
+Direct connection is still experimental and not tested in all the possible scenarios, but will hopefully become the standard mode in an future, because the approach is simpler and avoids most the synchronized mode defects (e.g. misalignments).
```

Comment:
```suggestion
Direct connection approach is simpler and avoids most the synchronized mode defects (e.g. misalignments).
```

Reply:
@offtherailz it not yet experimental, we have been using it in production for many years.